dsl/properties/scripts/ImportMicroservices.groovy

```diff
@@ -166,6 +185,28 @@ public class ImportMicroservices extends EFClient {
             version = getImageVersion(serviceConfig.image)
             url = getRegistryUri(serviceConfig.image)
         }
+      
+        //network
+        def networks = []
+
+        def networkList = []
+        if(serviceConfig.networks) {
+            serviceConfig.networks.each { key, value ->
+                networkList.push(key)
+            }
+        }
+        efService.service.networkList = networkList
```

Comment:
Are you sure this works? Networks are the part of mapping, not the service itself.